src/data/behandlerdialog/behandlerdialogQueryHooks.ts

```diff
@@ -0,0 +1,25 @@
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { ISBEHANDLERDIALOG_ROOT } from "@/apiConstants";
+import { post } from "@/api/axios";
+import { MeldingTilBehandlerDTO } from "@/data/behandlerdialog/behandlerdialogTypes";
+
+export const behandlerdialogQueryKeys = {
+  behandlerdialog: (personident: string) => ["behandlerdialog", personident],
+};
+
+export const useMeldingTilBehandler = () => {
+  const personident = useValgtPersonident();
+  const queryClient = useQueryClient();
+  const path = `${ISBEHANDLERDIALOG_ROOT}/melding`;
+  const postSendDialogmelding = (meldingTilBehandler: MeldingTilBehandlerDTO) =>
+    post(path, meldingTilBehandler, personident);
+
+  return useMutation(postSendDialogmelding, {
+    onSuccess: () => {
+      return queryClient.invalidateQueries(
```

Comment:
Når man bruker `return` her betyr det at mutation er `loading` helt til den querien er oppdatert (dvs GET er gjort på nytt) (ref: https://tkdodo.eu/blog/mastering-mutations-in-react-query#awaited-promises). Mulig det holder at knappen er loading til POST er ferdig? (i så fall kan `return` droppes, bare kall `invalidateQueries()`)

Reply:
Bra catch! Jeg kan endre dette, enig i at det er greit om den loader POST bare